### PR TITLE
Fix documentation and spacing inconsistencies

### PR DESCRIPTION
**Pull request by nicoleepp**
Fix some inconsistencies I came across in the documentation and spacing. Also added some type information that will be continued in #9.

Also renamed `start_worker` and `register_worker` for issue #11

### CODE REVIEW

File: src/client.jl [hunk 1]
```diff
@@ -9,11 +9,11 @@ results. Should only be used directly for advanced workflows. See [`DaskExecutor
 instead for normal usage.
 
 # Fields
-- `nodes::Dict{String, DispatchNode}`: maps keys to their dispatcher `DispatchNode`
+- `nodes::Dict{String, DispatchNode}`: maps previously submitted byte keys
```
Comment (iamed2) [inline src/client.jl]: What's a "byte key"?

File: src/worker.jl [hunk 2]
```diff
@@ -535,7 +535,7 @@ Add a task to the worker's list of tasks to be computed.
 - `who_has::Dict`: Map of dependent keys and the addresses of the workers that have them.
 - `nbytes::Dict`: Map of the number of bytes of the dependent key's data.
 - `duration::String`: The estimated computation cost of the given key. Defaults to "0.5".
-- `resource_restrictions::Dict`: Resources required by a task. Should always be an empty Dict.
+- `resource_restrictions::Dict`: Resources required. Should always be an empty Dict.
```
Comment (iamed2) [inline src/worker.jl]: `Resources required` by what for what?

File: docs/src/pages/manual.md [hunk 3]
```diff
@@ -60,8 +60,8 @@ using ResultTypes
 data = [1, 2, 3]
 
 ctx = @dispatch_context begin
-    a = @op 1 + 2
-	x = @op a + 3
+	a = @op 1 + 2
+ 	x = @op a + 3
```
Comment (iamed2) [inline docs/src/pages/manual.md]: This seems to have been all mixed up. Should use spaces and not tabs, and this block should have consistent indentation.

File: src/client.jl (hunk 1 above)
Comment (nicoleepp) [inline src/client.jl]: a key in byte form, will simplify this to just `keys`

File: docs/src/pages/manual.md (hunk 3 above)
Comment (nicoleepp) [inline docs/src/pages/manual.md]: Yea, indentation was off that I was trying to fix. Will try again

File: src/client.jl (hunk 1 above)
Comment (nicoleepp) [inline src/client.jl]: I will also rename this from`nodes` to `keys` when I update this Dictionary to a Set as part of another MR. (This docstring is outdated: nodes is actually a `Dict{Array{UInt8, 1}, Void}` right now, will just update it to reflect the correct current type and will fix the Types and name in the other PR)

**Comment by nicoleepp** [comment]
I removed the type parts for a future PR. Anything else this needs?

**Comment by iamed2** [comment]
It looks good but check out how the removal of dispatch context affects the indentation changes at https://github.com/invenia/DaskDistributedDispatcher.jl/pull/12/files#diff-25132558fdc7f9f2a23e2877d62ac347R64

**Comment by iamed2** [comment]
(you can do that after merge if you want)

**Pull request merged**